src/analysis/topology/extremeRays/lrs/extremePathways.m

```diff
@@ -99,20 +99,16 @@
 lrsInputHalfspace(A, D, filename, positivity, inequality, a, d, f, sh);
 
 % pause(eps)
-if isunix
-    [status, result] = system('which lrs');
-    if ~isempty(strfind(result, '/lrs'))
-        % call lrs and wait until extreme pathways have been calculated
-        systemCallText = ['lrs ' pwd filesep filename '_' suffix '.ine > ' pwd filesep filename '_' suffix '.ext'];
-        [status, result] = system(systemCallText);
-        if status == 1
-            error(['lsr failed on file ', pwd filesep filename '_' suffix '.ine']);
-        end
-    else
-        error('lrs is not installed on your machine or not in the path. Refer to http://cgm.cs.mcgill.ca/~avis/C/lrs.html to install lrs.');
+[status, result] = system('which lrs');
+if ~isempty(strfind(result, '/lrs'))
```

Comment:
May you use here `filesep`?

Reply:
Nope. 
`which` (also on windows) returns / as file separator. I was puzzled as well, but see the comments in the other files where this is looked up.

Reply:
Interesting ... may you put that as a note?